test/studies/sieve/sieve.chpl

```diff
@@ -0,0 +1,33 @@
+/*
+ * Sieve of Eratosthenes Example
+ *
+ * This example program finds prime numbers up to a limit using the
+ * sieve of Eratosthenes.
+ */
+
+use List;
+
+config var limit : uint = 1000; // maximum number to include in the sieve
+
+proc main() {
+  // list of confirmed primes
+  var primes : list(uint);
+  // range of potential prime numbers
+  var potentialPrimes = 2..limit;
```

Comment:
I'd probably inline this definition into the uses (that gets the code to be a tiny bit clearer for me and also removes any potential for confusion between `potentialPrime` vs `potentialPrimes`). But if you want to keep it as a separate variable I think you should declare it as `const potentialPrimes = 2..limit`.